infra/k8s/README-do-doks.md

```diff
@@ -55,13 +55,13 @@ kubectl apply -f sidecar.yaml
 ## Run a Testground testplan
```

Comment:
This README is going away - see next PR.

Reply:
Could we rename this to "README-deprecated.md" and add a comment?

Reply:
Currently there are a few setups that we experimented with, with only one of them having a working sidecar:
1. AWS EKS (the Terraform playbooks) - one-click install for a k8s cluster on AWS EKS
2. Digital Ocean DOKS setup - manual install on Digital Ocean though the Web interface
3. AWS `kops` with 2 Flannel networks
4. AWS `kops` with Weave/Flannel networks

In the follow up PR I am removing everything but 4. because it is the only setup that has a working sidecar, and looks like we are going down that path.

---

So we can either keep various README-s for 1,2,3 in the codebase as `deprecated` (maybe in a `deprecated` folder), or we can remove them. I opted for the second (removing them in the next PR) because they really don't pass our requirements, and would like to keep our docs to the point.

I could keep them as you suggest as `README-deprecated.md` just wanted to share my thought process :)

Reply:
Got it. Sorry, I read "do-doks" as some kind of weird way to say "do the docs". We can probably leave it this way for now.